sdk/util/rotation.h

```diff
@@ -126,6 +126,20 @@ class Rotation {
   // Multiply a Rotation and a Vector to get a Vector.
   VectorType operator*(const VectorType& v) const;
 
+  // Functions that return the Yaw, Pitch and Roll angle from the current 
+  // value of quat_. 
+  //
+  // @details     Yaw: rotation around the y-axis. Range: from -π to π
```

Comment:
Consider defining the range with mathematical notation to improve its understanding, and using M_PI (math macro) to avoid using non-ASCII characters, e.g.:

Yaw: rotation around the y-axis. Range: [-M_PI, PI)

See https://en.wikipedia.org/wiki/Interval_(mathematics)#Notations_for_intervals